internal/services/account_service.go

```diff
@@ -34,7 +38,16 @@ func NewAccountService(models *data.Models, metricsService metrics.MetricsServic
 	}, nil
 }
 
+// isValidStellarAddress validates that the address is either a valid Stellar public key or contract address
+func isValidStellarAddress(address string) bool {
+	return strkey.IsValidEd25519PublicKey(address) || strkey.IsValidContractAddress(address)
+}
```

Comment:
Can we rename to `isValidAddress`? "Stellar address" implies the G account format